apps/jobs/views.py

```diff
@@ -745,3 +745,40 @@ def update_submission(request, challenge_pk):
     submission.save()
     response_data = {'success': 'Submission result has been successfully updated'}
     return Response(response_data, status=status.HTTP_200_OK)
+
+
+@api_view(['GET', ])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def get_challenge_submissions_by_status(request, challenge_pk):
```

Comment:
Please change the name to `get_submissions_for_challenge()` and add the status as a query param. 

Also, it would be nice if we can start using django filter backend for filtering. See https://django-filter.readthedocs.io/en/latest/guide/rest_framework.html for more info.

Reply:
Since we need this feature urgently, I've added it in the list and will add it later during GSoC.